charts/scalardb/values.yaml

```
@@ -0,0 +1,111 @@
+# Default values for scalardb.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# nameOverride -- String to partially override scalardb.fullname template (will maintain the release name)
+nameOverride: ""
+# fullnameOverride -- String to fully override scalardb.fullname template
+fullnameOverride: ""
+
+scalardb:
+  # scalardb.replicaCount -- Default values for number of replicas.
+  replicaCount: 3
+
+  storageConfiguration:
+    # scalardb.storageConfiguration.contactPoints -- The database contanct point such as a hostname of Cassandra or a URL of Cosmos DB account.
+    contactPoints: cassandra
+    # scalardb.storageConfiguration.contactPort -- The database port number.
+    contactPort: 9042
+    # scalardb.storageConfiguration.username -- The username of the database. For Cosmos DB please leave blank. For Dynamo DB please specify key id here.
+    username: cassandra
+    # scalardb.storageConfiguration.password -- The password of the database. For Cosmos DB, Dynamo DB please specify a secret key here.
+    password: cassandra
+    # scalardb.storageConfiguration.storage -- Storage implementation. Either cassandra or cosmos or dynamo or jdbc can be set.
+    storage: cassandra
+
+  image:
+    # scalardb.image.repository -- Docker image reposiory of Scalar DB server.
+    repository: ghcr.io/scalar-labs/scalardb-server
+    # scalardb.image.pullPolicy Specify a image pulling policy.
+    pullPolicy: IfNotPresent
+    # scalardb.image.tag -- Docker tag of the image.
+    tag: 3.1.0
+
+  # scalardb.imagePullSecrets -- Optionally specify an array of imagePullSecrets. Secrets must be manually created in the namespace.
+  imagePullSecrets: []
+
+  strategy:
+    rollingUpdate:
+      # scalardb.strategy.rollingUpdate -- The number of pods that can be created above the desired amount of pods during an update
+      maxSurge: 25%
+      # scalardb.strategy.rollingUpdate -- The number of pods that can be unavailable during the update process
+      maxUnavailable: 25%
+    # scalardb.strategy.type -- New pods are added gradually, and old pods are terminated gradually, e.g: Recreate or RollingUpdate
+    type: RollingUpdate
+
+  service:
+    # scalardb.service.type -- service types in kubernetes.
+    type: ClusterIP
+    ports:
+      scalardb-server:
```

It's a small detail, but maybe `scalardb` will do.
```suggestion
      scalardb:
```